MapboxSearch/ui/src/main/java/com/mapbox/search/ui/extensions/DeviceLocatioonProvider.kt

```diff
@@ -0,0 +1,22 @@
+package com.mapbox.search.ui.extensions
+
+import com.mapbox.common.location.LocationProvider
+import com.mapbox.geojson.Point
+import com.mapbox.search.base.utils.extension.toPoint
+import com.mapbox.search.common.DistanceCalculator
+
+/**
+ * TODO
+ *
+ */
+public fun LocationProvider.userDistanceTo(destination: Point, callback: (Double?) -> Unit) {
```

Comment:
I'm not sure whether this function should be public. I'd make it internal.

Anyway, api reference should be fixed.